examples/amp-story/bookendv1.json

```diff
@@ -15,6 +15,20 @@
       "title": "This is an example article",
       "url": "http://example.com/article.html",
       "image": "http://placehold.it/256x128"
+    },
+    {
+      "type": "cta-link-pair",
+      "text1": "sign up",
```

Comment:
we should add a spec option to indicate "this is an amp link", so that we can do AMP-to-AMP linking where possible (adding `rel="amphtml"` to the `<a>` tag; see #12496).  this probably also applies to the other linkable components, so I think it's fine to address this in a separate PR.

Reply:
Cool, created #15261 to do in a follow-up PR.